src/processing.rs

```diff
@@ -43,13 +59,15 @@ impl Processing {
             match File::open(file) {
                 Ok(mut file) => {
                     let mut buffer = String::new();
-                    // TODO: Error handling
-                    file.read_to_string(&mut buffer).unwrap();
-                    debug!("{}", to_html(&buffer));
+                    match file.read_to_string(&mut buffer) {
+                        Ok(_) => debug!("{}", to_html(&buffer)),
+                        Err(e) => return WikiError::new(ErrorType::BufferStringifyFailed, &e.to_string()),
```

Comment:
I think you can avoid these matches and simply use the `?` operator here and below.